nav/load_waypoints/scripts/navigate_waypoints.py

```diff
@@ -121,8 +127,18 @@ def get_pose_from_gps(self, longitude, latitude, frame, pose_test_var = None):
 
         return p_in_frame
 
-        
-        
+    def get_pose_from_xy(self, x, y):
+
+        # Convert (x,y) coordinate to pose
+        # (x,y) coordinate follows the coordinate system used in spawn.launch
+
+        xy_pose = PoseStamped()
+        xy_pose.header.frame_id = 'odom' # not sure what the proper frame is
+        xy_pose.pose.position.x = y - 0 # subtract starting y-coordinate
```

Comment:
What are these hardcoded params? Could you make a note (we need to know if we need to adjust them for the actual comp)

Reply:
removed the xy stuff